packages/build/src/plugins_core/functions/index.js

```diff
@@ -88,6 +91,7 @@ const zipFunctionsAndLogResults = async ({
 const coreCommand = async function ({
   constants: {
     INTERNAL_FUNCTIONS_SRC: relativeInternalFunctionsSrc,
+    IS_LOCAL: isRunningLocally,
```

Comment:
@ehmicky Can you advise on whether we should run this when `mode !== "buildbot"` or when `mode === "cli"`?

Reply:
It seems to me it should also run when `mode` is `require` (programmatic usage)? In that case, this would be when `mode !== "buildbot"`, which means `IS_LOCAL` is correct here.

Reply:
Thanks!